compiler/ballerina-lang/src/main/java/io/ballerina/projects/CompletionManager.java

```diff
@@ -165,9 +170,12 @@ private TypeSymbol getRawType(TypeSymbol typeDescriptor) {
         return typeDescriptor;
     }
 
-    private boolean isInServiceBodyNodeContext(CompletionContext context, Node referenceNode) {
+    private boolean isInServiceBodyNodeContext(CompletionContext context,
+                                               Node referenceNode) {
         Optional<LinePosition> cursorPosition = context.cursorPosition();
-        if (referenceNode.kind() != SyntaxKind.SERVICE_DECLARATION || cursorPosition.isEmpty()) {
+        Node nodeAtCursor = context.nodeAtCursor();
```

Comment:
Can't we define this right before this used?

Reply:
Done